docs/stransmit.md

```diff
@@ -80,55 +99,115 @@ Be careful with options being specified together with having standard
+necessary, as the IGMP addresses are recognized by their mask.
+
+
+For sending to a multicast group:
+
+* The **iptos** parameter designates the Type-Of-Service (TOS) field for
+outgoing packets via `IP_TOS` socket option.
+
+* The **ttl** parameter will set time-to-live value for outgoing packets via
+`IP_MULTICAST_TTL` socket options.
+
+* The **adapter** parameter can be used to specify the adapter to be set
+through `IP_MULTICAST_IF` option to override the default device used for
+sending
+
+
+For receiving from a multicast group:
+
+* The **adapter** parameter can be used to specify the adapter through which
+the given multicast group can be reached (it's used to bind the socket)
+
+* The **source** parameter enforces the use of `IP_ADD_SOURCE_MEMBERSHIP`
+instead of `IP_ADD_MEMBERSHIP` and the value is set to `imr_sourceaddr` field.
+
+Explanations for the symbols and terms used above can be found in POSIX
+manual pages, like `ip(7)` and on Microsoft docs pages under `IPPROTO_IP`.
 
-If you use multicast IP address:
-* For reading, need extra `@` character before the *host* part so that the application subscribes to the multicast group before reading
-* The *host* part designates the multicast group (also as a resolvable name)
-* The *port* designates the port in the multicast group
-* The **adapter** parameter can be used to specify the adapter through which the given multicast group can be reached
 
 Medium: SRT
 -----------
 
 Most important about SRT is that it can be either input or output and in
 both these cases it can work in listener, caller and rendezvous mode. SRT
 also handles several parameters special way, in addition to standard SRT
-options that can be set through the parameters:
-
-    srt://HOST:PORT?PARAM1=VALUE&PARAM2=VALUE...
+options that can be set through the parameters.
 
 SRT can be connected using one of three connection modes:
 
-- **caller**: the "agent" (this application) sends the connection request to the peer, which must be **listener**, and this way it initiates the connection.
-- **listener**: the "agent" waits for being contacted by any peer **caller** (note that a listener can accept multiple callers, but *srt-live-transmit* does not use this possibility - after the first connected one, it no longer accepts new connections).
-- **rendezvous**: A one-to-one only connection where both parties are equivalent and both connect to one another simultaneously. Whoever happened to start first (or succeeded to punch through the firewall) is meant to have initiated the connection.
+- **caller**: the "agent" (this application) sends the connection request to
+  the peer, which must be **listener**, and this way it initiates the
+connection.
+
+- **listener**: the "agent" waits for being contacted by any peer **caller**
+  (note that a listener can accept multiple callers, but *srt-live-transmit*
+does not use this possibility - after the first connected one, it no longer
+accepts new connections).
```

Comment:
- **listener**: the "agent" waits to be contacted by any peer **caller**.
Note that a listener can accept multiple callers, but *srt-live-transmit*
does not use this ability; after the first connection, it no longer
accepts new connections.